sys/linux/init_vusb_ids.go

```diff
@@ -0,0 +1,7957 @@
+// AUTOGENERATED FILE
```

Comment:
Leave some reference to what generated this and how to re-generate this (docs/linux/external_fuzzing_usb.md?).
I can link usb_device_id to generateUsbDeviceID and then find usbIds and this file, but then traces lost and it's not part of make generate.

Reply:
Done.